### DISCUSSION TITLE
How to view the code of a browser extension?

### DISCUSSION THREAD

**Discussion by 2Abendsegler** [2Abendsegler/GClh#2466] [Questions] [answered]
How to view the code of a browser extension?
I'm currently concerned with the following extension:
`https://addons.mozilla.org/de/firefox/addon/statshunters`

@capoaira 
Have you an idea?

**Comment by capoaira** [comment]
Ne, leider nicht :(

**Reply by 2Abendsegler** [reply, marked as answer]
Durch Informationen von Herr-Ma ist das nun klar.

Die Add ons von Firefox sollten in folgendem Verzeichnis stehen: 
`C:\Users\[user]\AppData\Roaming\Mozilla\Firefox\Profiles\[profile]\extensions`
Die dort stehenden xpi Dateien sind zip Archive der Add ons.
xpi Datei kopieren, in zip umbenennen und dann kann man sich die Bestandteile des Add ons ansehen.
